listenstore/listenstore/listenstore.py

```diff
@@ -1,42 +1,36 @@
 # coding=utf-8
-from __future__ import division, absolute_import
+from __future__ import absolute_import
 from __future__ import print_function, unicode_literals
 import logging
 import calendar
-import ujson
```

Comment:
Earlier alastairp mentioned that they were having issues with `ujson` in AB. So i thought it was better to remove it altogether. Also, the json's what we are dealing with, are not quite big, so the performance gain would not be that significant.